packages/lib/src/js/charts/abstractChart.js

```diff
@@ -80,6 +83,7 @@ export default class AbstractChart {
    * @param {Function} [tooltipFunction] function that receives a data object and returns the string displayed as tooltip.
    * @param {Array} [legend] names of the sub-arrays of data, used as legend labels.
    * @param {String | Object} [legendTarget] DOM node to which the legend should be mounted.
+   * @param {Object} [point={ color: null, radius: 3, fillOpacity: 1, strokeWidth: 0, strokeColor: null }] point object specifying color, radius, fillOpacity, strokeWidth and strokeColor.
```

Comment:
This can be split up to multiple lines, see https://github.com/documentationjs/documentation/blob/master/docs/RECIPES.md#destructuring-parameters